tasks/hyrax_dev.rake

```diff
@@ -30,6 +30,25 @@ task :i18n_sorter do
   end
 end
 
+namespace :engine_cart do
+  # This generate task should only add its action to an existing generate task
+  raise 'engine_cart:generate task should already be defined' unless Rake::Task.task_defined?('engine_cart:generate')
+  task :generate do |_task|
```

Comment:
Where is the original EC:generate task being run in the redeclared one? (Or is that not necessary due to Rake :sparkles:?)

Reply:
[Rake :sparkles:.](https://ruby.github.io/rake/doc/rakefile_rdoc.html#label-Multiple+Definitions)
A task's block is appended to a list of existing action blocks.